public/index.html

```diff
@@ -28,7 +28,6 @@
     <title>SharedBox</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
Esta linea es bueno dejarla. Las aplicaciones creadas con React solo funcionan cuando el usuario tiene javascript activado en su navegador. Si por alguna razón el usuario tiene JS desactivado se le va a mostrar ese mensaje

Reply:
listo, back to the code